tests/func_test.py

```diff
@@ -32,6 +32,37 @@ def delete(self):
         self.db_conn
 
 
+class FoobarHandler(requesthandlers.APIHandler):
+    """ No input_schema_use_defaults defined, so it will raise errors
+    normally despites default key is declared on schema.
```

Comment:
"No ``input_schema_use_defaults`` defined, so it will raise errors normally despite default key being declared in the schema."